apps/qubit/modules/digitalobject/actions/editAction.class.php

```diff
@@ -241,6 +298,30 @@ public function processForm()
       $representation->save();
     }
 
+    // Upload new video track files
+    foreach ($this->videoTracks as $usageId => $videoTrack)
+    {
+      if (null !== $uf = $this->form->getValue("trackFile_$usageId"))
+      {
+        $lang = $this->form->getValue("lang_$usageId");
+        $uploadedTracks[$usageId] = array($uf, $lang);
+      }
+    }
+
+    foreach ($uploadedTracks as $usageId => $uploadTrack)
+    {
+      $content = file_get_contents($uploadTrack[0]->getTempName());
```

Comment:
Even if this files are small and shouldn't have a big memory footprint, `QubitAsset` allows to pass directly a path to avoid saving the file contents in memory. For example ...

https://github.com/artefactual/atom/blob/qa/2.x/lib/task/digitalobject/digitalObjectLoadTask.class.php#L371

I'm not completely sure this will work with the uploaded files in the tmp folder, but I think we should check if it does.

Reply:
I'm essentially using the same method that is currently being used for representations. I think that this is touching on an issue above my little PR, so I'll defer to you folks to do this.